jest.config.mjs

```diff
@@ -1,5 +1,10 @@
 import nextJest from 'next/jest.js';
 
+const date = new Date();
+const timestamp = `${date.getFullYear()}-${
+  date.getMonth() + 1
+}-${date.getDate()}-${date.getHours()}-${date.getMinutes()}`;
+
```

Comment:
not sure what the intention is for having date in the coverage report. 
we don't commit coverage report in our code base.

Reply:
everything else looks good.

Reply:
> not sure what the intention is for having date in the coverage report. we don't commit coverage report in our code base.

So we don't need to save every coverage report? Do we just need to save the current/most recent one which gets overwritten with new one when we run tests again?